packages/flutter/lib/src/material/dropdown_menu.dart

```diff
@@ -490,6 +490,7 @@ class _DropdownMenuState<T> extends State<DropdownMenu<T>> {
   double? leadingPadding;
   bool _menuHasEnabledItem = false;
   TextEditingController? _localTextEditingController;
+  FocusNode? _focusNode;
```

Comment:
When this focus node is created internally, it should also be disposed by this widget